service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -279,6 +223,46 @@ public void notifyUnpaidOrder(Order order) {
         }
     }
 
+    private Long getAmountToPay(Order order) {
+        Long bonuses = order.getPointsToUse() == null ? 0L : order.getPointsToUse().longValue();
+        Long certificates = order.getCertificates() == null ? 0L
+            : order.getCertificates().stream()
+                .map(Certificate::getPoints)
+                .reduce(0, Integer::sum)
+                .longValue();
+
+        Long paidAmount = order.getPayment() == null ? 0L
+            : order.getPayment().stream()
+                .filter(payment -> payment.getPaymentStatus().equals(PaymentStatus.PAID))
+                .map(payment -> payment.getAmount() / 100)
```

Comment:
please do not use [magic numbers](https://www.javatpoint.com/magic-number-in-java#:~:text=If%20the%20single%20digit%20is,325%20is%20a%20magic%20number.)